setup.py

```diff
@@ -34,7 +34,7 @@ def read_long_description() -> str:
     python_requires=">3.6",
     install_requires=[
         "attrs>=18.2.0",
-        "black>=21.4b0",
+        "black[jupyter]>=21.4b0",
```

Comment:
I don't think this works if there is already an older version of `black` in your environment. 

```
...
"black>=21.9b0",
"black[jupyter]",
...
```

seems to do the job, although it's not particularly elegant.

Reply:
Huh, it doesn't work even with `-U` flag? Either way is fine by me.

Reply:
No, because for an older version of black it doesn't know what `[jupyter]` is. I don't think `-U` would help because that would say "upgrade `black-nb`" rather than "upgrade `black`"

Reply:
Ok, I'll close the PR.

Reply:
Actually I think it's fine. If you just change `black[jupyter]>=21.4b0` to `black[jupyter]>=21.9b0` I'll go ahead and merge. Sorry for the back and forwards.

Reply:
Ok, got it.